README.md

```diff
@@ -2,7 +2,7 @@
 
 # Porter's Go Vanity Server
 
-Porter uses a vanity import, for example `get.porter.sh/porter` instead of `github.com/deislabs/porter`, so that in the future when Porter moves to another GitHub organization, it doesn't cause problems and break mixins.
+Porter uses a vanity import, for example `get.porter.sh/porter` instead of `github.com/getporter/porter`, so that in the future when Porter moves to another GitHub organization, it doesn't cause problems and break mixins.
```

Comment:
Since we have just done the move and don't plan on doing another, let's remove that old wording to avoid giving people ideas.

```suggestion
Porter uses a vanity import, for example `get.porter.sh/porter` instead of `github.com/getporter/porter`, so that our go package imports aren't tied to our source control.
```